python/src/aac/plugins/first_party/lsp_server/providers/rename_provider.py

```diff
@@ -65,38 +66,37 @@ def get_rename_edits(
 
     def _get_rename_edits(self, symbol: str, new_name: str) -> dict[str, list[TextEdit]]:
         """Returns the list of text edits based on the selected symbol's type."""
-        if not symbol:
-            return None
-
         language_context = self.language_server.language_context
 
-        name = remove_list_type_indicator(symbol).strip(":")
-        symbol_types = get_possible_symbol_types(name, language_context)
+        # Strip any ':' that accompany yaml keys from the incoming replacement text
+        sanitized_new_name = new_name.strip(":")
+        text_to_replace = remove_list_type_indicator(symbol).strip(":")
```

Comment:
Slight improvement but we should probably follow the `remove_list_type_indicator` pattern and put the `strip(":")` into a simple helper function, as well.

Reply:
I adjusted the logic so that we're only stripping `new_name`. I don't foresee the need to do a lot of stripping like this -- it's necessary because of how VSCode's rename function incorporates the colon in the request. So, TLDR: I don't see it as necessary to provide a function to wrap `.strip(':')` here.